packages/hdwallet-ledger-webusb/src/transport.ts

```diff
@@ -14,6 +14,7 @@ import Btc from '@ledgerhq/hw-app-btc'
 import getAppAndVersion from '@ledgerhq/live-common/lib/hw/getAppAndVersion'
 import getDeviceInfo from '@ledgerhq/live-common/lib/hw/getDeviceInfo'
 import openApp from '@ledgerhq/live-common/lib/hw/openApp'
+import { listApps } from "@ledgerhq/live-common/lib/apps/hw"
```

Comment:
What is the difference between this `apps/hw` implementation and the `/hw` implementation here: https://github.com/LedgerHQ/ledger-live-common/blob/master/src/hw/listApps.js

Reply:
Good questions. @ryankeepkey mentioned this. The one at  https://github.com/LedgerHQ/ledger-live-common/blob/master/src/hw/listApps.js uses the native transport vs the one we are using we pass it our transport. Not sure what the difference is beyond that but something to keep in mind the one we are using here doesn't work.

Reply:
I am pretty sure both of them pass the transport. That is the way we set up those calls. We wouldn't need to do the `getDeviceInfo` with the `/hw` either. The rest of the utility functions we are using are also in `/hw`. I was surprised to even see the other implementation of `listApps` in `apps/hw`.